test-coverage.txt

```diff
@@ -1,85 +1,117 @@
-  index.js                  |        0 |        0 |        0 |        0 |                   |
- pages/Home                 |       80 |      100 |       50 |       80 |                   |
-  Home.page.jsx             |    77.78 |      100 |       50 |    77.78 |             17,23 |
-  Home.styled.js            |      100 |      100 |      100 |      100 |                   |
-  index.js                  |        0 |        0 |        0 |        0 |                   |
- pages/NotFound             |      100 |      100 |      100 |      100 |                   |
-  NotFound.page.jsx         |      100 |      100 |      100 |      100 |                   |
-  index.js                  |        0 |        0 |        0 |        0 |                   |
- pages/VideoDetail          |    90.91 |       50 |       75 |    90.91 |                   |
-  VideoDetail.page.jsx      |    84.62 |       50 |       75 |    84.62 |             27,37 |
-  VideoDetail.styled.js     |      100 |      100 |      100 |      100 |                   |
-  index.js                  |        0 |        0 |        0 |        0 |                   |
- router                     |      100 |      100 |      100 |      100 |                   |
-  index.js                  |      100 |      100 |      100 |      100 |                   |
- service                    |      100 |      100 |      100 |      100 |                   |
-  mockData.js               |      100 |      100 |      100 |      100 |                   |
- utils                      |      100 |      100 |      100 |      100 |                   |
-  constants.js              |      100 |      100 |      100 |      100 |                   |
- utils/hooks                |    85.71 |      100 |    66.67 |    85.71 |                   |
-  useService.js             |    85.71 |      100 |    66.67 |    85.71 |                 9 |
- utils/store                |    65.31 |       10 |    71.43 |    63.04 |                   |
-  actions.js                |    57.14 |      100 |     62.5 |       50 |... 28,33,35,39,43 |
-  context.js                |      100 |      100 |      100 |      100 |                   |
-  providers.js              |      100 |      100 |      100 |      100 |                   |
-  reducers.js               |    42.86 |       10 |       50 |    42.86 |... 40,51,53,58,63 |
-----------------------------|----------|----------|----------|----------|-------------------|
+ PASS  src/components/Toogle/ToogleTheme.test.js
+-------------------------------|----------|----------|----------|----------|-------------------|
+File                           |  % Stmts | % Branch |  % Funcs |  % Lines | Uncovered Line #s |
+-------------------------------|----------|----------|----------|----------|-------------------|
+All files                      |       70 |    32.53 |    60.38 |    70.42 |                   |
```

Comment:
It's interesting that the branch percentage went up but all the other percentages went down